src/core/FormField.ts

```diff
@@ -5,13 +5,16 @@ import get from 'lodash.get';
 import set from 'lodash.set';
 
 export class FormField<Values> {
-  private readonly _schema: yup.Schema<Values>;
+  readonly field: string | symbol;
+
+  private readonly _schema?: yup.Schema<Values>;
 
   private _errors: FormError<Values> = {};
 
   private _touched: { [K in keyof Values]?: FormTouched<Values[K]> } = {};
 
-  constructor(schema: yup.Schema<Values>) {
+  constructor(field: string | symbol, schema?: yup.Schema<Values>) {
```

Comment:
```suggestion
  constructor(public readonly field: string | symbol, private readonly _schema?: yup.Schema<Values>) {
```

No need to declare and initial fields twice.